2022-01-15 14:47:42,358 INFO    MainThread:156138 [wandb_setup.py:_flush():71] setting env: {'project': 'exercise_14', 'run_group': 'dev'}
2022-01-15 14:47:42,358 WARNING MainThread:156138 [wandb_setup.py:_flush():71] could not save program above cwd: /home/marc/.conda/envs/mlflow-acbfa4ee4aa9be3158264a605e4861dd8a78622b/bin/pytest
2022-01-15 14:47:42,358 INFO    MainThread:156138 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-15 14:47:42,358 INFO    MainThread:156138 [wandb_init.py:_log_setup():371] Logging user logs to /home/marc/development/python/udacity_course/genre_classification/starter/check_data/wandb/run-20220115_144742-23orgmji/logs/debug.log
2022-01-15 14:47:42,359 INFO    MainThread:156138 [wandb_init.py:_log_setup():372] Logging internal logs to /home/marc/development/python/udacity_course/genre_classification/starter/check_data/wandb/run-20220115_144742-23orgmji/logs/debug-internal.log
2022-01-15 14:47:42,359 INFO    MainThread:156138 [wandb_init.py:init():404] calling init triggers
2022-01-15 14:47:42,360 INFO    MainThread:156138 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-15 14:47:42,360 INFO    MainThread:156138 [wandb_init.py:init():460] starting backend
2022-01-15 14:47:42,360 INFO    MainThread:156138 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-15 14:47:42,374 INFO    MainThread:156138 [backend.py:ensure_launched():216] starting backend process...
2022-01-15 14:47:42,376 INFO    MainThread:156138 [backend.py:ensure_launched():221] started backend process with pid: 156175
2022-01-15 14:47:42,377 INFO    MainThread:156138 [wandb_init.py:init():469] backend started and connected
2022-01-15 14:47:42,378 INFO    MainThread:156138 [wandb_init.py:init():533] updated telemetry
2022-01-15 14:47:42,389 INFO    MainThread:156138 [wandb_init.py:init():563] communicating current version
2022-01-15 14:47:42,870 INFO    MainThread:156138 [wandb_init.py:init():568] got version response 
2022-01-15 14:47:42,870 INFO    MainThread:156138 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-15 14:47:43,176 INFO    MainThread:156138 [wandb_init.py:init():606] starting run threads in backend
2022-01-15 14:47:43,727 INFO    MainThread:156138 [wandb_run.py:_console_start():1810] atexit reg
2022-01-15 14:47:43,728 INFO    MainThread:156138 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-15 14:47:43,728 INFO    MainThread:156138 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-15 14:47:43,729 INFO    MainThread:156138 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-15 14:47:43,729 INFO    MainThread:156138 [wandb_init.py:init():633] run started, returning control to user process
2022-01-15 14:47:46,515 INFO    MainThread:156138 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-01-15 14:47:46,515 INFO    MainThread:156138 [wandb_run.py:_restore():1752] restore
2022-01-15 14:47:49,237 INFO    MainThread:156138 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1021
  total_bytes: 1021
}

2022-01-15 14:47:49,506 INFO    MainThread:156138 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1021
  total_bytes: 1021
}

2022-01-15 14:47:50,190 INFO    MainThread:156138 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1021
  total_bytes: 5529
}

2022-01-15 14:47:50,292 INFO    MainThread:156138 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1021
  total_bytes: 5529
}

2022-01-15 14:47:50,395 INFO    MainThread:156138 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1021
  total_bytes: 5529
}

2022-01-15 14:47:50,497 INFO    MainThread:156138 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1021
  total_bytes: 5529
}

2022-01-15 14:47:50,599 INFO    MainThread:156138 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1021
  total_bytes: 5529
}

2022-01-15 14:47:50,700 INFO    MainThread:156138 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1021
  total_bytes: 5529
}

2022-01-15 14:47:50,803 INFO    MainThread:156138 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5529
  total_bytes: 5529
}

2022-01-15 14:47:50,905 INFO    MainThread:156138 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5529
  total_bytes: 5529
}

2022-01-15 14:47:51,006 INFO    MainThread:156138 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5529
  total_bytes: 5529
}

2022-01-15 14:47:51,108 INFO    MainThread:156138 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5529
  total_bytes: 5529
}

2022-01-15 14:47:51,212 INFO    MainThread:156138 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5529
  total_bytes: 5529
}

2022-01-15 14:47:51,504 INFO    MainThread:156138 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5529
  total_bytes: 5529
}

2022-01-15 14:47:51,988 INFO    MainThread:156138 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5529
  total_bytes: 5529
}
local_info {
}

2022-01-15 14:47:53,062 INFO    MainThread:156138 [wandb_run.py:_append_files():2180] logging synced files
